src/test/java/org/rulez/demokracia/pdengine/ChoiceDeleteValidationTest.java

```diff
@@ -58,8 +58,7 @@ public void invalid_adminKey_is_rejected() throws ReportedException {
 	public void proper_voteId_choiceId_and_adminKey_does_delete_choice() throws ReportedException {
 		String choiceId = voteManager.addChoice(adminInfo.adminKey, adminInfo.voteId, "choice1", "user");
 		String voteId = adminInfo.voteId;
-		String result = voteManager.deleteChoice(adminInfo.voteId, choiceId, adminInfo.adminKey);
-		assertEquals("OK", result);
```

Comment:
deleteChoice returns "OK"
https://adadocs.demokracia.rulez.org/PDEngine/edemo/master/index.html#3044c21c-7606-4098-8176-9dd5b93c20ad
What is that throw below?

Reply:
Ok, I understand the throw now.